### DISCUSSION TITLE
Administrative Tasks Examples

### DISCUSSION THREAD

**Discussion by VinceHache** [PowerShell/PowerShell#18115] [General]
What are a few examples of administrative tasks that can be done from a PowerShell script that are not for Active Directory or Azure?

**Comment by kasini3000** [comment]
Welcome to the powershell remote management project under my name.
1 I also wrote the script psed.ps1, similar to SED for linux.
2 There is a re-package of cwrsync.
3 There are win, linux batch password change tools.

I also wrote a TUI interface linux command alias tool.

The database client is very important to powershell, I encapsulated several database clients, they call the .net library to complete the database connection.
